src/components/Tool/FixSendNewsletter/FixSendNewsletter.test.tsx

```diff
@@ -167,8 +194,8 @@ describe('FixSendNewsletter', () => {
         expect(mockEnqueue).toHaveBeenCalledWith('Newsletter updated!', {
           variant: 'success',
         });
-        expect(queryByText(name)).not.toBeInTheDocument();
-        expect(queryByText(otherName)).toBeInTheDocument();
+        expect(queryByText(firstContactName)).not.toBeInTheDocument();
```

Comment:
Could you please also check that the `firstContactName` was on the page before the confirm was clicked?